src/Handler.php

```diff
@@ -56,6 +56,27 @@ public function __construct(Composer $composer, IOInterface $io) {
     $this->composer = $composer;
     $this->io = $io;
     $this->progress = TRUE;
+
+    // Pre-load all of our sources so that we do not run up
+    // against problems in `composer update` operations.
+    $this->manualLoad();
+  }
+
+  protected function manualLoad() {
+    $src_dir = __DIR__;
+
+    $classes = [
+      'CommandProvider',
+      'DrupalScaffoldCommand',
+      'FileFetcher',
+      'PrestissimoFileFetcher',
+    ];
+
+    foreach ($classes as $src) {
+      if (!class_exists('\\DrupalComposer\\DrupalScaffold\\' . $src)) {
```

Comment:
Is the a particular reason for the double backslash notation?

Reply:
The double backslash is necessary at the end; otherwise you get `\'`, which escapes the last single-quote. I made the rest doubled up for consistency. This could be changed if you prefer it the other way.

Reply:
Ok, makes sense. Thanks!